lib/pip.js

```diff
@@ -62,7 +63,9 @@ function generateRequirementsFile(
 ) {
   if (
     options.usePoetry &&
-    fse.existsSync(path.join(servicePath, 'pyproject.toml'))
+    fse.existsSync(
+      path.join(servicePath, 'pyproject.toml') && isPoetryProject(servicePath)
```

Comment:
I believe this breaks poetry if it is actually in usee
Shouldn't this be:
`fse.existsSync(path.join(servicePath, 'pyproject.toml')) && isPoetryProject(servicePath)`
not 
`fse.existsSync(path.join(servicePath, 'pyproject.toml') && isPoetryProject(servicePath))`

and same below?

causing this issue: https://github.com/UnitedIncome/serverless-python-requirements/issues/497

Reply:
Parentheses are tricky little buggers.

Reply:
(I'm going to fix this and merge.)